geocode_array/Bing.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+import pprint
+import time
+import urllib.parse
+
+from geocode_array.Geocoder import Geocoder
+from geocode_array import REQUEST_HEADER_DICT, REQUEST_TRIES, REQUEST_DELAY
+
+
+def _form_rev_bing_request(request_base_url, request_url_args, proxy_url) -> urllib.request.Request:
+        request_string = f"{request_base_url}{request_url_args}"
```

Comment:
doesn't want any character between the base and request urls

Reply:
I think the easier solution will be to make this string template that is passed in:

```python
def _form_rev_bing_request(request_base_url, request_url_args, proxy_url, request_template) -> urllib.request.Request:
        request_string = request_template.format(request_base_url, request_url_args)
```

For geocoder (and hence, its children), then just make the template argument default to `{}?{}`, and override it for the Bing one?

Gets you out of mostly repeating the two functions.

Reply:
Yeah, that's a better idea, only overwriting `_reverse_geocode` now